### PR TITLE
DAT-97: rename "link" by "fileName"

### PR DESCRIPTION
**Pull request by jhoffman-ms**
Rename Link property in the InvoiceListItem by FileName:

image

### CODE REVIEW

File: Billing.API/Models/InvoiceListItem.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public InvoiceListItem(string product, string accountId, DateTimeOffset date, st
             Date = date;
             Currency = currency;
             Amount = amount;
-            Link = link;
+            FileName = fileName;
```
Comment (andresmoschini) [inline Billing.API/Models/InvoiceListItem.cs:24]: If I am not wrong, generally, _"filename"_ is treated as a single word (see [Wikipedia](https://en.wikipedia.org/wiki/Filename))

```suggestion
            Filename = filename;
```

File: README.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Authorization: Bearer {{token-jwt}}
             "currency": "ARS",
             "amount": 128,
             // There will probably be more data
-            "link": "//path_on_server/filename.ext"
+            "fileName": "filename.ext"
```
Comment (andresmoschini) [inline README.md:36]:
```suggestion
            "filename": "filename.ext"
```

File: Billing.API/Models/InvoiceListItem.cs (hunk 1 above)
Comment (jhoffman-ms) [reply Billing.API/Models/InvoiceListItem.cs:24]: good, I will change

File: README.md (hunk 2 above)
Comment (jhoffman-ms) [reply README.md:36]: yes, the same as above.

**Pull request merged**